cli/commands/helpers/subcommand.go

```diff
@@ -0,0 +1,63 @@
+package helpers
```

Comment:
I feel this code would be better namespaced in the command package, as "helpers" has less meaning. https://github.com/golang/go/wiki/CodeReviewComments#package-names